examples/cpp/systemd_socket_activation/test.sh

```diff
@@ -54,17 +49,49 @@ ReusePort=true
 WantedBy=sockets.target
 EOF
 
-systemctl daemon-reload
-systemctl enable sdsockact.socket
-systemctl start sdsockact.socket
+    # reload after adding units
+    systemctl --user daemon-reload
+
+    ## starting at boot is not necessary for testing
```

Comment:
this can be deleted.

Reply:
Ok, i will remove it and update the branch